src/component/FolderMain.js

```diff
@@ -0,0 +1,78 @@
+import { FolderCard } from "./FolderCard";
+import "../css/main.css";
+import { UserFolderList } from "./UserFolderList";
+import pen from "../asset/pen.svg";
+import deleteImg from "../asset/delete.svg";
+import share from "../asset/share.svg";
+import { useState } from "react";
+
+function FolderMain({ folderInfo, userFolderList }) {
+  const [selectedFolderInfo, setSelectedFolderInfo] = useState([]);
+  const [selectedButton, setSelectedButton] = useState(false);
```

Comment:
- true | false의 boolean 상태라면 이름도 is-나 has- 로 드러내는 게 좋습니다
- 사용하는 곳을 같이 보니까 "선택된 폴더가 있는지" 를 의미하는 군요. 그럼 그 뜻을 그대로 작성하면 됩니다. 글 처럼 읽으면서 맥락을 따라갈 수 있는게 좋습니다
- 데이터 중심으로, Button같은 (쉽게 바뀔 수 있는)UI 요소 보다는 데이터의 이름을 따서 이름을 짓는게 좋습니다
 
```suggestion
  const [isFolderSelected, setIsFolderSelected] = useState(false);
```